src/app/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Header } from '../widgets/header';
+import './index.scss';
+import { Routing } from '../pages';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
use our ordering. From global to inner modules

Reply:
import { BrowserRouter } from 'react-router-dom';
import { Routing } from '../pages';
import { Header } from '../widgets/header';
import './index.scss';
Like this?